modules/rosa-sts-aws-requirements-creating-association.adoc

```diff
@@ -7,15 +7,23 @@
 [id="rosa-associating-account_{context}"]
 = Linking your AWS account
 
-You link your AWS account using the `rosa` CLI.
+You can link your AWS account to existing IAM roles by using the `rosa` CLI.
 
 .Prerequisites
 
 * You have an AWS account.
 * You are using {cluster-manager-url} to create clusters.
 * You have the permissions required to install AWS account-wide roles. See the "Additional resources" of this section for more information.
 * You have installed and configured the latest AWS (`aws`) and ROSA (`rosa`) CLIs on your installation host.
-* You have created your `ocm-role` and `user-role` IAM roles.
+* You have created your `ocm-role` and `user-role` IAM roles, but have not yet linked them to your AWS account. You can check whether your IAM roles are already linked by running the following commands:
++
+[source,terminal]
+----
+$ rosa list ocm-role
```

Comment:
I think the [OpenShift docs guidance](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output) is only one command per code block, but it's unclear to me if that would also apply to a prerequisite with a code block.

Reply:
I can update this so that it's two separate code blocks to comply with this, thankyou!